### PR TITLE
Fix linting issues in app/views/articles

### PR DESCRIPTION
**Pull request by Glennmen**
## What type of PR is this? (check all applicable)

- [X] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Documentation Update

## Description
Was able to fix all linting errors.

~~Still open issues: `111 error(s) were found in ERB files`~~

~~Linting errors that I couldn't solve:~~

~~Most errors where in this file: `app/views/articles/_v2_form.html.erb`~~
~~These 3 errors where mentioned here:~~
```
expected space after attribute value
expected whitespace, '>' or '=' after attribute name
No space detected where there should be a single space.
```

~~One of the errors also thrown in: `app/views/articles/_markdown_form.html.erb`~~
~~`expected whitespace, '>', attribute name or value`~~

~~This line was throwing an error but wasn't sure how to correctly handle it:
`app/views/articles/_markdown_form.html.erb:15`~~

~~Also wasn't sure what to do this: `app/views/articles/_video_player.html.erb:8`~~
```
Style/FormatStringToken: Prefer annotated tokens (like `%<foo>s`) over unannotated tokens (like `%s`).
```

An additional suggestion, I had to manually fix this because my Jetbrains Reformat Code kept reverting it. Not sure we want the spaces or could do without. For example here: `app/views/articles/_markdown_form.html.erb:1`
```
Layout/SpaceInsideHashLiteralBraces: Space inside { missing.
```

If any additional info is needed just ask 😉 
This was bigger than I expected, I hope that the other folders will contain less changes.

## Related Tickets & Documents
#1842 

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [X] no documentation needed

### CODE REVIEW

File: app/views/articles/_video_player.html.erb [hunk 1]
```diff
@@ -5,27 +5,27 @@
     <meta itemprop="description" content="<%= @article.description %>" />
     <meta itemprop="thumbnailUrl" content="<%= cloudinary(article.video_thumbnail_url, 880) %>" />
     <meta itemprop="contentUrl" content="<%= article.video_source_url %>" />
-    <meta itemprop="duration" content="<%= "PT%sM%sS" % article.video_duration_in_minutes.split(':') %>" />
+    <meta itemprop="duration" content="<%= "PT%sM%sS" % article.video_duration_in_minutes.split(":") %>" />
```
Comment (lightalloy) [inline app/views/articles/_video_player.html.erb]: The issue can be fixed by:
```erb
<% minutes, seconds = @article.video_duration_in_minutes.split(":") %>
<meta itemprop="duration" content="<%= format("PT%<minutes>sM%<seconds>sS", minutes: minutes, seconds: seconds) %>" />
```
If anyone has an idea how to fix it prettier, please share.
@Glennmen please, also pull from master to get the new `erblint` config, or it'll complain on "unused variables".

File: app/views/articles/_v2_form.html.erb [hunk 2]
```diff
@@ -1,40 +1,60 @@
...
+      <input class="articleform__title" type="text" value="<%= @article.title %>" placeholder="Title" />
+      <div class="articleform__detailfields">
+        <div class="articleform__tagswrapper">
+          <textarea type="text" class="articleform__tags" placeholder="tags"><%= @article.cached_tag_list %></textarea>
+        </div>
+        <button class="articleform__detailsButton articleform__detailsButton--image">
+          <img src="<%= asset_path "image-upload.svg" %>"> IMAGES
+        </button>
+        <button class="articleform__detailsButton articleform__detailsButton--moreconfig">
+          <img src="<%= asset_path "three-dots.svg" %>"></button>
```
[the remark this answers is not in the document]
Comment (Glennmen) [reply app/views/articles/_v2_form.html.erb]: I added it for consistency but it is actually not required in HTML: https://stackoverflow.com/a/23890795/7446162

**Comment by Glennmen** [comment]
I can't make a code review of my own PR but something I wasn't sure about:
File: `app/views/articles/_video_player.html.erb` line 1.
`<div temscope itemtype="http://schema.org/VideoObject" class="video-player-header">`
Should `temscope` not be `itemscope` ?

File: app/views/articles/_v2_form.html.erb (hunk 2 above)
[the remark this answers is not in the document]
Comment (lightalloy) [reply app/views/articles/_v2_form.html.erb]: Agree. Actually, I've disabled the "Self-closing" cop myself and forgot about it. Sorry )

**Comment by lightalloy** [comment]
@Glennmen I believe it should be `itemscope`.

**Comment by lightalloy** [comment]
As for the spaces inside the hashes, I think they should be added, cause we have `Layout/SpaceInsideHashLiteralBraces` cop enabled both for the `*.rb` and the views code. 
Probably you had to fix it manually, cause your IDE is configured differently.

**Comment by Glennmen** [comment]
> As for the spaces inside the hashes, I think they should be added, cause we have `Layout/SpaceInsideHashLiteralBraces` cop enabled both for the `*.rb` and the views code.
> Probably you had to fix it manually, cause your IDE is configured differently.

For anyone having the same "issue" as me in the future and is using Jetbrains auto format
Preferences -> Editor -> Code Style -> Ruby -> Other -> Enable `Spaces around curly braces in hashes`

File: app/views/articles/_sticky_nav.html.erb [hunk 3]
```diff
@@ -1,32 +1,33 @@
...
   }
-  .primary-sticky-nav-author button{
+
+  .primary-sticky-nav-author button {
     background-color: <%= user_colors(@actor)[:bg] %> !important;
     color: <%= user_colors(@actor)[:text] %> !important;
-    <% if user_colors(@actor)[:bg].downcase == "#ffffff" %>
-      border: 1px solid <%= user_colors(@actor)[:text] %> !important;
-    <% end %>
+  <% if user_colors(@actor)[:bg].downcase == "#ffffff" %> border: 1px solid <%= user_colors(@actor)[:text] %> !important;
```
[the remark this answers is not in the document]
Comment (Glennmen) [reply app/views/articles/_sticky_nav.html.erb]: This actually might be my IDE that did that and not related to linting. Will look into that.

**Comment by lightalloy** [comment]
@Glennmen could you resolve the conflict?

**Comment by maestromac** [comment]
Sorry @Glennmen can you resolve merge conflict one last time?

**Comment by Glennmen** [comment]
@maestromac I merged master again, but was getting pre-commit errors in other files now (from a different merge).

`/app/javascript/article-form/elements/publishToggle.jsx`
`app/javascript/article-form/articleForm.jsx`
`app/javascript/article-form/actions.js`
`app/javascript/article-form/__tests__/articleForm.test.jsx`

76 problems (73 errors, 3 warnings)

**Comment by maestromac** [comment]
@Glennmen oh darn. When you merge from master, after you finish resolving the conflict, the commit to resolve merge conflict should be done with `--no-verify` because you aren't responsible for those additional changes.

**Comment by Glennmen** [comment]
@maestromac I already did 😉

File: app/views/articles/_single_story.html.erb [hunk 4]
```diff
@@ -1,65 +1,65 @@
...
-      <a href="<%= story.user.path %>"><%= story.user.name %>・<%= story.readable_publish_date %></a>
-      
-    </h4>
-    <div class="tags">
-      <% story.cached_tag_list_array.each do |tag| %>
-        <a href="/t/<%= tag %>"><span class="tag">#<%= tag %></span></a>
+  <% end %>
+  <% if story.organization && !@organization_article_index %>
+    <div class="article-organization-headline">
+      <a href="/<%= story.organization.slug %>" class="article-organization-headline-inner"><img src="<%= organization.profile_image_90 %>"><%= organization.name %>
```
Comment (maestromac) [inline app/views/articles/_single_story.html.erb]:
```suggestion
      <a href="/<%= story.organization.slug %>" class="article-organization-headline-inner"><img src="<%= organization.profile_image_90 %>"><%= story.organization.name %>
```

Comment (Glennmen) [reply app/views/articles/_single_story.html.erb]: Good catch, doesn't the part before that also have to be changed? Can you add that to your suggestion so I can commit it on Github.

Comment (maestromac) [reply app/views/articles/_single_story.html.erb]: Oh nice catch! updated.

**Comment by Glennmen** [comment]
I have no idea why `app/javascript/article-form/articleForm.jsx` is in `Changed files`, I think prettier kicked in on pre-commit hook and might have changed some things. This shouldn't happen if it was committed correctly in the first place.

**Pull request merged**